test/mapmatch.cc

```diff
@@ -597,13 +597,13 @@ void test_topk_frontage_alternate() {
   tyr::actor_t actor(conf, true);
   auto matched = json_to_pt(actor.trace_attributes(
       R"({"costing":"auto","best_paths":2,"shape_match":"map_snap","shape":[
-           {"lat":52.07956040090567,"lon":5.138160288333893,"accuracy":10},
-           {"lat":52.07957358807355,"lon":5.138508975505829,"accuracy":10},
-           {"lat":52.07959666560798,"lon":5.138905942440034,"accuracy":10},
-           {"lat":52.0796213915245,"lon":5.139262676239015,"accuracy":10},
-           {"lat":52.079637875461195,"lon":5.139581859111787,"accuracy":10},
-           {"lat":52.07964776582031,"lon":5.139828622341157,"accuracy":10},
-           {"lat":52.07985600778458,"lon":5.1404121782302865,"accuracy":10}]})"));
+           {"lat":52.07956040090567,"lon":5.138160288333893,"accuracy":10,"time":2},
+           {"lat":52.07957358807355,"lon":5.138508975505829,"accuracy":10,"time":4},
+           {"lat":52.07959666560798,"lon":5.138905942440034,"accuracy":10,"time":6},
+           {"lat":52.0796213915245,"lon":5.139262676239015,"accuracy":10,"time":8},
+           {"lat":52.079637875461195,"lon":5.139581859111787,"accuracy":10,"time":10},
+           {"lat":52.07964776582031,"lon":5.139828622341157,"accuracy":10,"time":12},
+           {"lat":52.07985600778458,"lon":5.1404121782302865,"accuracy":10,"time":14}]})"));
```

Comment:
I see you added `time` to all these test cases. Should we keep one or two test cases without timestamps to test the default case?

Reply:
Is it possible also to add an assertion on the returned duration? At least make sure it's non-zero / non-negative?